### ISSUE TITLE
Additional information missing IDs in principles

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#442]
In the additional information section, I did not find the ID markup like I found in the rest of the file.

**Comment by gregoriopellegrino** [comment]
Yes, it was a choice, we decided to keep the Additional accessibility information out, because it can potentially carry many different types of terms and it is not aligned between ONIX and EPUB Accessibility Metadata...

**Comment by GeorgeKerscher** [comment]
I believe we have all IDs now.
